src/hooks/useResetScroll.tsx

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
-import { useLocation, useNavigate } from '@reach/router';
+import { useLocation, useNavigate } from 'react-router-dom';
 
 export const useResetScroll = () => {
   const navigate = useNavigate();
-  const { href, state } = useLocation();
+  const { state, pathname } = useLocation();
 
-  const updateState = React.useCallback(() => {
-    navigate(href, {
+  /*const updateState = React.useCallback(() => {
```

Comment:
there are two options for this function, using useCallback and useLayoutEffect, or using useEffect. If we don't use useEffect, there are a whole bunch of warnings printed out onto the console about how navigate needs to be called in a useEffect. But other than that, I don't notice any difference in execution

Reply:
useEffect seems fine. Go ahead and remove the commented out code.